resources/geocoding/hy/374.txt

```diff
@@ -59,11 +59,11 @@
 37423492|Մարտիրոսյան
 37423497|Փոքր Վեդի
 37423498|Տափերական
-374235|Արտաշատ/Այգեզարդ/Դալար/Քաղցրաշեն/Մխչյան/Շահումյան
+374235|Արտաշատ/Այգեվազ/Դալար/Քաղցրաշեն/Մխչյան/Շահումյան
```

Comment:
The English file is still saying Aygezard for this, and I don't get results for "Այգեվազ" on a Google search.
Please provide more information about this change, such as online or local information about Aygevaz and that this is the name preferred over Aygezard (see our [contribution guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence)).